### PR TITLE
new resource `azurerm_spring_cloud_api_portal`

### PR DESCRIPTION
**Pull request by ms-henglu**
image

### CODE REVIEW

File: website/docs/r/spring_cloud_api_portal.html.markdown [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+* `name` - (Required) The name which should be used for this Spring Cloud API Portal. Changing this forces a new Spring Cloud API Portal to be created.
+
+* `spring_cloud_service_id` - (Required) The ID of the Spring Cloud Service. Changing this forces a new Spring Cloud API Portal to be created.
+
+---
+
+* `gateway_ids` - (Optional) Specifies a list of Spring Cloud Gateway.
+
+* `https_only` - (Optional) is only https is allowed?
```
Comment (katbyte) [inline website/docs/r/spring_cloud_api_portal.html.markdown:65]: shoudl this be?
```suggestion
* `https_only_enabled` - (Optional) is only https is allowed?
```

**Review by katbyte — CHANGES_REQUESTED** [review]
test failures:
image

**Comment by ms-henglu** [comment]
@katbyte - I've fixed the testcases, and renamed the field, please take another look, thanks!
image

**Review by katbyte — APPROVED** [review]
Thanks @ms-henglu - LGTM 🍾

**Pull request merged**